src/Foundatio.Jobs.Commands/Foundatio.Jobs.Commands.csproj

```diff
@@ -4,6 +4,11 @@
     <TargetFrameworks>net46</TargetFrameworks>
     <PackageTags>Distributed;Jobs;NETSTANDARD;Core</PackageTags>
   </PropertyGroup>
+  <PropertyGroup Label="Signing">
+    <SignAssembly>true</SignAssembly>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+    <AssemblyOriginatorKeyFile>..\Exceptionless.snk</AssemblyOriginatorKeyFile>
```

Comment:
We need to copy the strong name file over, I'm currently not sure how the build is passing (it should fail if the strong named file doesn't exist).